packages/backend/src/notifications/firebaseConfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Te dane nigdy nie powinny pojawic sie w git. Jeśli dane nie sa fake'owe to każdy kto przekopiuje je może sie polaczyc do naszego firebase jako admin. Trzeba sprawdzić czy można ten prywatny klucz zmienic. Jak nie to trzeba utworzyć nowe konto.

Reply:
Czemu masz tu zmiany backendowe?